ods_ci/run_robot_test.sh

```diff
@@ -290,7 +301,8 @@ fi
 if [[ ${SKIP_INSTALL} -eq 0 ]]; then
   poetry install
 fi
-source $(poetry env info --path)/bin/activate
+# shellcheck disable=SC1091
```

Comment:
why?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC1091

Reply:
what happens without? sorry I don't know much about shellchek

Reply:
I think it just prints out a warning here in GitHub. Annoying but not the end of the world.

Reply:
Yeah - this is to disable the warning, since the warning is irrelevant for us.

Also, per configuration - if there are more than 10 warnings reported by shellcheck, then relevant CI job is marked as failed, which isn't nice for our PRs.

Reply:
thanks!